wolfssl/wolfcrypt/aes.h

```diff
@@ -198,6 +201,7 @@ struct Aes {
 #if defined(WOLF_CRYPTO_CB) || (defined(WOLFSSL_DEVCRYPTO) && \
     (defined(WOLFSSL_DEVCRYPTO_AES) || defined(WOLFSSL_DEVCRYPTO_CBC)))
     word32 devKey[AES_MAX_KEY_SIZE/WOLFSSL_BIT_SIZE/sizeof(word32)]; /* raw key */
+    int    keySet;
```

Comment:
The `keySet` is Octeon specific. Can we add a `HAVE_CAVIUM_OCTEON_SYNC` check around this?

Reply:
Yes.